### DISCUSSION TITLE
Makine a /broadcast command.

### DISCUSSION THREAD

**Discussion by RespDev** [Minestom/Minestom#2246] [Help] [answered]
I need to know how to have command input that is an infinite amount of words.

I tried this:

`var broadcastArgument = ArgumentType.String("broadcastArgument");`

But that code did not work for infinite amount of words as command input.

**Comment by TheMode** [comment, marked as answer]
You can use `ArgumentType.StringArray`

**Reply by RespDev** [reply]
Thanks!
